docs/tigerbeetle.md

```diff
@@ -0,0 +1,19 @@
+# Rafiki and Tigerbeetle
+
+TigerBeetle is a distributed financial accounting database designed for mission critical safety and performance. Since Rafiki implements the [Interledger Protocol](./glossary.md#interledger-protocol), which splits payments into packets, we need a high performing and safe database to store all that payment data. For detailed information on Tigerbeetle, including its consesus mechanism and its limitations, visit the official Tigerbeetle [documentation](https://docs.tigerbeetle.com/) and [blog](https://tigerbeetle.com/blog/).
```

Comment:
```suggestion
TigerBeetle is a distributed financial accounting database designed for mission critical safety and performance. Since Rafiki implements the [Interledger Protocol](./glossary.md#interledger-protocol), which splits payments into packets, we need a high performing and safe database to store all that payment data. For detailed information on Tigerbeetle, including its consensus mechanism and its limitations, visit the official Tigerbeetle [documentation](https://docs.tigerbeetle.com/) and [blog](https://tigerbeetle.com/blog/).
```